.travis.yml

```diff
@@ -1,5 +1,4 @@
 language: node_js
 node_js:
-  - 7
-  - 6
-  - 4
```

Comment:
I'm very excited about node 8 => mostly for the improved V8 engine performance, but also for async/await. That said, is it time to deprecate node 4 - 6? Node 8 only became LTS last week. Though I guess since most Hapi modules seem to have moved it's probably time?

Reply:
That's why this change is a breaking semver change. People using older versions of Node can still use the current release. I should make this more clearer in the README.